rfcs/text/semantic-release.md

```diff
@@ -0,0 +1,257 @@
+- Start Date: 2019-11-28
+
```

Comment:
Maybe add a TOC here?
```markdown
# Table of contents

- [Summary](#summary)
- [Basic example](#basic-example)
- [Motivation](#motivation)
- [Detailed design](#detailed-design)
  - [Subject](#subject)
  - [Body](#body)
  - [Footer](#footer)
  - [Reverting a Release](#reverting-a-release)
  - [Release Notes](#release-notes)
  - [Maintenance Releases](#maintenance-releases)
- [Drawbacks](#drawbacks)
- [Alternatives](#alternatives)
- [Adoption strategy](#adoption-strategy)
- [How we teach this](#how-we-teach-this)
  - [Learning Git](#learning-git)
    - [Atomic Commits](#atomic-commits)
      - [Don’t push your work until you’re happy with it](#dont-push-your-work-until-youre-happy-with-it)
      - [Examples](#examples)
```

Reply:
Added in eccfc511dc6ef008b96aad63f4f6af2461eece42